src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -3711,10 +3723,10 @@ std::optional<UInt64> StorageReplicatedMergeTree::totalRowsByPartitionPredicate(
     return res;
```

Comment:
We also need to add context here.

Reply:
The context already exists in the method without change.

Reply:
I mean the `context` needs to be passed into `foreachCommittedParts` . And there are other build errors https://clickhouse-test-reports.s3.yandex.net/16309/feaab260d8e591371febab76b8e26c726111d8aa/fast_test/runlog.out.log

Reply:
Ok, I have fixed all build errors now.